graylog2-server/src/main/java/org/graylog/plugins/views/search/QueryResult.java

```diff
@@ -83,14 +83,21 @@ public static Builder create() {
                     .executionStats(QueryExecutionStats.empty());
         }
 
+        @JsonProperty
         public abstract Builder query(Query query);
 
+        @JsonProperty("execution_stats")
         public abstract Builder executionStats(QueryExecutionStats stats);
 
+        @JsonProperty("search_types")
         public abstract Builder searchTypes(Map<String, SearchType.Result> results);
 
+        @JsonProperty("errors")
+        @Nullable
+        @JsonInclude(JsonInclude.Include.NON_ABSENT)
```

Comment:
We usually don't use `NON_ABSENT`. It makes it harder to discover the API when payload fields only exist in certain situations.

Reply:
@bernd: The annotation was used for the class before this PR already. The field will still be part of the OpenAPI spec, so I think it is okay regarding discoverability.

Reply:
Alright. :+1: